app/controllers/v0/reference_data_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module V0
+  class ReferenceDataController < ApplicationController
```

Comment:
I think this needs to be namespaced somehow, both the class and the route. I mean states and countries ought to be safe but thinking of this in holistic vets-api terms, are all these endpoints broadly applicable across the API? If not, they should be given some context.

Reply:
I completely agree and I don't know the answer.  I did not know this `RDS` existed until it was pitched as an easy API for `vets-api` to integrate with using JWT.  _**How**_ and _**If**_ these endpoints will be used by anyone is still a mystery to me.  It's worth mentioning that `GET /countries` & `GET /states` [_already exist_ in the `address_controller`](https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/controllers/v0/addresses_controller.rb#L19) - and yes, they return identical data just formatted slightly different.

Reply:
Armin Burkholder from EVSS pasted me the readme from the Github where `RDS` is hosted since I don't have access:
```
# What is this repository for?
This repository contains the VetsAPI Reference Data microservice for benefits and compensation applications.
# Overview - VetsAPI Reference Data
The service operations offer lists of data from authoritative sources that can be used by applications to conform to the expectations of other benefits and compensation services.
## Operations offered
(note that "/referencedata" is likely to change)
1. /referencedata/v1/countries<br>
  A list of country names, confroming to the federal naming conventions of the U.S. Treasury department.
2. /referencedata/v1/states<br>
  A list of state abbreviations, conforming to the federal conventions of the U.S. Treasury department.
3. /referencedatav1/intakesites<br>
  A list of VA Intake sites.
4. /referencedata/v1/treatmentcenter<br>
  A list of VA Treatment Centers
5. /referencedata/v1/disabilities<br>
  The list of VA defined disabilities for benefits and compensation purposes.
# Service Consumers - How to use this API
See the swagger for the microservice at http://{server.domain}:8762/api/vetsapi-refdata/swagger-ui.html
# Requests and Contributions
If your team has requests for changes or additions to this service, please use the GitHub Issues tab.
If your team would like to contribute to the project, please fork the repo to make your changes, and [Create a pull request from the fork](https://help.github.com/articles/creating-a-pull-request-from-a-fork/).
## Contributors - How to set up
See the wiki tab.
```

Reply:
```
Bill Ryan [3:05 PM] 
im still a bit confused: is this API different than what we've been calling the "PCIU Address Service"? which includes the endpoints:
- `/wss-pciu-services-web/rest/pciuServices/v1/countries`
- `/wss-pciu-services-web/rest/pciuServices/v1/states`

[3:05] 
vs the RDS defined:
- `/referencedata/v1/countries`
- `/referencedata/v1/states`

Armin Burkholder [3:05 PM] 
Yes, in theory they should return the same lists

[3:06] 
Some of this is just trying to move out of legacy

Bill Ryan [3:08] 
next question(s) though: is there a desire to use one over the other in `prod` / `staging`?

Armin Burkholder [3:09 PM] 
Good question.  This is our first leap into the Ascent platform. It is a billion times better.  If it was me, I would use refdata.
```

Reply:
@raquelromano-gov might have more context